docs/guide/data_types/maps.rst

```diff
@@ -478,9 +478,10 @@ very easily when initializing the normalization variable. For example::
 This clips out many of the brightest pixels. If you'd like to see what areas of
 your images got clipped set the following values::
 
-    >>> cmap = cmap.plot_settings['cmap']  # doctest: +SKIP
+    >>> cmap = map.cmap  # doctest: +SKIP
     >>> cmap.set_over('red', 1.0)  # doctest: +SKIP
     >>> cmap.set_under('green', 1.0)  # doctest: +SKIP
+    >>> map.cmap = cmap   # doctest: +SKIP
```

Comment:
```suggestion
```
This line doesn't work – you can't set this property – and even if it did work, it'd be unnecessary because you've already modified the colormap directly.

Reply:
Ahh, good catch.